test/CMakeLists.txt

```diff
@@ -29,9 +29,10 @@ link_libraries(
         Qt5::Multimedia
         Qt5::UiTools
         Qt5::Widgets)
+		
+find_package(Lua51 REQUIRED)
 
-
-include_directories("${CMAKE_CURRENT_SOURCE_DIR}/../src")
+include_directories("${CMAKE_CURRENT_SOURCE_DIR}/../src" "${CMAKE_CURRENT_SOURCE_DIR}/../3rdparty/edbee-lib/edbee-lib" "/usr/include/lua51")
```

Comment:
1. Hardcoding the `edbee-lib` include path here is wrong. See https://github.com/Mudlet/Mudlet/blob/development/src/CMakeLists.txt#L398 for the correct way to include those headers.
2. If you use `find_package` for lua (which is the right thing to do), hardcoding the include path here is wrong. See https://github.com/Mudlet/Mudlet/blob/development/src/CMakeLists.txt#L401 how the include should be handled.

Reply:
>Very late to the party
Better late than never, as a few versions have been released since then, I thought the patch would just be dropped as MXP to the extend I'd like to use is probably no prio.
I'll try your hints. All these are only for auto TEST being run. I've not been able to find the proper paths in the build system. Vadi wasn't able to hint me.  I assume I'll have to rejoin with the current devel path since some time passed.  I'll get back to you regarding this.

Reply:
Hello keneanung,

I had no luck rebuilding on Github, C-Style checks now bail out in countless Mudlet and internal QT library files I never touched. I have not the slightest clue how to fix this.

I'm outta here. Submitting these IMHO relatively simple patches to Mudlet needs much too much effort for me. At least twice as much of the time than I can afford. I saw countless issues with MXP that need to be fixed to comply with standards and this was only meant as a first test patch to make it barely work, and I'm already unable to keep track with build system changes and stuff.

I give up, sorry. I understood Mudlet's idea of what mudclient is and should be differs from mine and has thus different priorities - which is totally ok - , but I'm outta here. Sorry.. I'm done, I can't help it.

Good luck with your project.